src/components/NewLocationPage/LocationName/utils.ts

```diff
@@ -0,0 +1,22 @@
+import { State, County, MetroArea } from 'common/regions';
+
+export function inMultipleStates(region: MetroArea): boolean {
+  return region.stateCodes.includes('-');
+}
+
+export function getRegionName(region: State | County | MetroArea) {
```

Comment:
this type can just be `Region`, which encompasses all three. you can import it from `common/regions`